### PR TITLE
Create index.md

### PR DESCRIPTION
**Pull request by alexmyczko**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of why you think this topic or collection should be curated.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/gnustep/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+created_by: GNUstep developers
+display_name: GNUstep
+logo: gnustep.png
+released: 'January 1995'
```
Comment (kobr3n) [inline topics/gnustep/index.md]: create-octocat

**Comment by kobr3n** [comment]
https://github.com/alexmyczko/explore.git.patch

File: topics/gnustep/index.md (hunk 1 above)
Comment (aTOmQpLUs) [reply topics/gnustep/index.md]: 🙈🙉🙊

**Comment by aTOmQpLUs** [comment]
🙈🙉🙊🔊

**Comment by aTOmQpLUs** [comment]
🏴󠁧󠁢󠁳󠁣󠁴🏆🌏

**Comment by talsafran** [comment]
Hiya @alexmyczko 👋 just checking if you're still hoping to get these changes merged in? See @kenyonj's [message](https://github.com/github/explore/pull/1029#pullrequestreview-295096907) above, you'll need to get CI green before continuing.

Thank you so much!

File: topics/gnustep/index.md (hunk 1 above)
Comment (elizabethcone86) [reply topics/gnustep/index.md]: Hello

Comment (elizabethcone86) [reply topics/gnustep/index.md]:
```suggestion
released: 'January 1995'
```

File: topics/gnustep/index.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+created_by: GNUstep developers
+display_name: GNUstep
+logo: gnustep.png
```
Comment (elizabethcone86) [inline topics/gnustep/index.md]: IMG_1849

**Comment by kenyonj** [comment]
Please reopen this PR when ready. Thanks.

**Pull request closed without merging**